lua/buffon/config.lua

```diff
@@ -18,6 +18,7 @@ local default = {
       "gitcommit",
       "gitrebase",
     },
+    default_position = "top_right",
```

Comment:
That's a good proposal, but I suggest adding `default_position` to the `open` attribute. I know `open` might not be the most descriptibe name, but for now, everything related to the main windows is located tehre.

Reply:
You're right. My mistake. 

Thanks